app/components/provider_interface/application_choice_header_component.rb

```diff
@@ -77,7 +93,7 @@ def show_inset_text?
       flash.empty? && (
         respond_to_application? || deferred_offer_wizard_applicable? ||
         rejection_reason_required? || provider_cannot_respond?
-      )
+      ) || waiting_for_interview?
```

Comment:
Should this be inside the `()`?

Reply:
No as both the notification message and the component displaying the avaiable actions should be rendered together (see here: https://bat-design-history.netlify.app/manage-teacher-training-applications/interviews-iteration-3/#set-up-interview-success-message-on-the-interviews-page). Again, this was part of my PR o it's already in master as well.

However I do wonder why the box is only rendered for other actions when there is no flash message

Reply:
I think it's because we only want to render one box at a time

Reply:
I'll speak with Adam and see if it was added to the mockup in error.